registry.go

```diff
@@ -4,6 +4,7 @@ import (
 	"github.com/docker/distribution/context"
 	"github.com/docker/distribution/digest"
 	"github.com/docker/distribution/manifest"
+	storagedriver "github.com/docker/distribution/registry/storage/driver"
```

Comment:
This file should never be dependent on storage driver. This is the distribution interface and should never know about the driver.